### ISSUE TITLE
Security SIR Other Capability Warning

### ISSUE THREAD

**Issue by murrayalex** [ombegov/ITDB-schema#152]
Name: Alex Murray
Organization: Booz Allen, supporting eCPIC PMO
Urgency: Medium
Description of Issue: I am receiving the warnings that "Other" capabilities should not exceed 10% of the spending for the respective category when I attempt to add new capabilities through the monthly. At this point, I do not have any capabilities tagged as "Other" (neither on the ITDB already or in my request). It seems to only trigger when adding capabilities that are in a category that already has a record on the ITDB. This isn't critical, but will be a source of confusion for our users.

[Request and Response.txt](https://github.com/ombegov/ITDB-schema/files/1270940/Request.and.Response.txt)
[Get Response.txt](https://github.com/ombegov/ITDB-schema/files/1270941/Get.Response.txt)

**Comment by ojedanico** [comment]
Alex, 
We where unable to replicate this.  Based on the text files attached, it seems you are submitting one other capability (27), but we don't know where the 1.00001 value is coming from and don't know why there is a warning for capability 6 and for capability 25. is the value coming from the AITPS?

We where able to get this warning when submitting an "other" capability, like 27, but it makes sense because we are adding one capability of this category, so the amount submitted for "other" is 100% of the amount for that category.  We are not getting warnings outside of what is expected, and the amounts shown in the warning make sense for the amounts submitted.

Any additional information you can provide will be helpful for us to diagnose this issue.

**Comment by murrayalex** [comment]
Hey Nico,
I will retest this. I was seeing the issue when submitting new capabilities of the same category as capabilities already saved to the dashboard. None of the capabilities were the "Other", which is why I noticed the issue. If I can still recreate it, I will send over the files.

**Comment by murrayalex** [comment]
I am still seeing it, Nico. These attached requests and responses are for a new investment. The Annual Add shows the initial three capabilities that were added. I then attempted to add one more through the monthly operation, prompting the warnings regarding Other Detect Capabilities (of which this investment has none).
[Monthly Add Request.txt](https://github.com/ombegov/ITDB-schema/files/1278145/Monthly.Add.Request.txt)
[Annual Add.txt](https://github.com/ombegov/ITDB-schema/files/1278146/Annual.Add.txt)
